### PR TITLE
Glimmer 0.7.2

### PR DESCRIPTION
**Pull request by mixonic**
* Tweak Glimmer 0.3.10 to use a single helper instead of two.
* Add Glimmer 0.7.2

screen shot 2017-09-01 at 8 38 20 pm

### CODE REVIEW

File: glimmer-v0.7.2-keyed/config/targets.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+let browsers = [
+  'last 1 Chrome versions'
```
Comment (mixonic) [inline glimmer-v0.7.2-keyed/config/targets.js]: Re: https://github.com/krausest/js-framework-benchmark/issues/243 I pinned this to only support the most recent version of Chrome.

**Comment by krausest** [comment]
Thanks. I‘m on holidays this week - I‘ll merge it when I‘m back home.

**Comment by krausest** [comment]
Results are [here](https://rawgit.com/krausest/js-framework-benchmark/b6228535b2b54de7bfe02483411e8aacaccf2a1c/webdriver-ts-results/table.html). Nice improvement, but create rows still has room for improvements.

**Comment by mixonic** [comment]
Thanks for the re-run @krausest :-) there is definitely continuing room for improvement! The glimmer app libraries are being updated to work with the latest version of the VM now. I'll continue to push these updates to here so we can track improvement in this benchmark.

**Pull request merged**